rest/rest-server/src/main/java/org/ow2/proactive_grid_cloud_portal/rm/RMRestInterface.java

```diff
@@ -459,11 +459,18 @@ Object getNodeMBeanInfo(@HeaderParam("sessionid") String sessionId, @QueryParam(
      * without redundancy, in a friendly JSON format.
      *
      * @param sessionId a valid session
+     * @param nodeJmxUrl mbean server url
+     * @param objectName name of mbean
+     * @param attrs set of mbean attributes
      * @param range a String of 5 chars, one for each stat history source, indicating the time range to fetch
      *      for each source. Each char can be:<ul>
      *            <li>'a' 1 minute
      *            <li>'m' 10 minutes
+     *            <li>'n' 5 minutes
+     *            <li>'t' 30 minutes
      *            <li>'h' 1 hour
+     *            <li>'j' 2 hours
+     *            <li>'k' 4 hours
```

Comment:
@aminelouati I didn't understand from where u brought 'n' 't' 'j' 'k' parameters

Reply:
Don't bother about this comment. I understood why you added them.